superset/assets/src/visualizations/PlaySlider.jsx

```diff
@@ -4,9 +4,7 @@ import { Row, Col } from 'react-bootstrap';
 
 import Mousetrap from 'mousetrap';
 
-import 'bootstrap-slider/dist/css/bootstrap-slider.min.css';
-import ReactBootstrapSlider from 'react-bootstrap-slider';
-import './PlaySlider.css';
```

Comment:
This import should still be here, since it has more styles than were moved to `BootrapSliderWrapper`.

Reply:
Good catch!